Changes

```diff
@@ -16,10 +16,14 @@ Working version
   API when the old block is NULL.
   (Jacques-Henri Jourdan, review by Xavier Leroy)
 
-- #8920, #9238: New API for statistical memory profiling in Memprof.Gc. The
-  new version does no longer use ephemerons and allows registering
-  callbacks for promotion and deallocation of memory blocks.
+- #8920, #9238, #9239: New API for statistical memory profiling in
+  Memprof.Gc. The new version does no longer use ephemerons and allows
+  registering callbacks for promotion and deallocation of memory
+  blocks.
   The new API no longer gives the block tags to the allocation callback.
+  The Gc.Memprof.start function can now only be called if sampling is
+  not currently active, and Gc.Memprof.stop can now only be called if
+  sampling is currently active.
```

Comment:
I think this is probably too detailed for end-users (they know the API has changed so they will read the new .mli; the main difference is at the C level); it can stay, but it would be fine to just have the PR number at the top and nothing more.

Reply:
You're right.